multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/listeners/CreateOrUpdateServiceEndListener.java

```diff
@@ -34,7 +28,31 @@ public void notify(DelegateExecution execution) {
                                                        .singleResult();
         superExecutionId = superExecutionResult.getSuperExecutionId();
 
+        boolean isServiceUpdated = VariableHandling.get(execution, Variables.IS_SERVICE_UPDATED);
+        String serviceName = VariableHandling.get(execution, Variables.SERVICE_TO_PROCESS_NAME);
+        if (serviceName == null) {
+            throw new IllegalStateException("Unable to determine service update status.");
+        }
+
+        setServiceUpdatedInParentProcess(runtimeService, superExecutionId, isServiceUpdated, serviceName);
+        setDynamicResolvableParameterInParentProcess(execution, runtimeService, superExecutionId, serviceName);
+    }
+
+    private void setServiceUpdatedInParentProcess(RuntimeService runtimeService, String superExecutionId, boolean isServiceUpdated,
+                                                  String serviceName) {
+        String exportedVariableName = Constants.VAR_IS_SERVICE_UPDATED_VAR_PREFIX + serviceName;
+
         runtimeService.setVariable(superExecutionId, exportedVariableName, isServiceUpdated);
+    }
 
+    private void setDynamicResolvableParameterInParentProcess(DelegateExecution execution, RuntimeService runtimeService,
```

Comment:
would this make sense as a string map serialized at once as opposed to different variables? (maybe not due to parallel processes?)

Reply:
Due to parallel processes, it can't be used single flowable variable.